src/ReCaptcha/RequestMethod/Socket.php

```diff
@@ -47,7 +47,7 @@ class Socket
      */
     public function fsockopen($hostname, $port = -1, &$errno = 0, &$errstr = '', $timeout = null)
     {
-        $this->handle = fsockopen($hostname, $port, $errno, $errstr, (is_null($timeout) ? ini_get("default_socket_timeout") : $timeout));
+        $this->handle = fsockopen($hostname, $port, $errno, $errstr, (empty($timeout) ? ini_get("default_socket_timeout") : $timeout));
```

Comment:
I'd prefer `!isset()` here and on #L68 as I think that makes the intent more explicit.